test_scripts/API/PerformAudioPassThru/025_ATF_P_PerformAudioPassThru_UI_SUCCESS_TTS_success_false_true.lua

```diff
@@ -0,0 +1,191 @@
+	{ result_code = "CHAR_LIMIT_EXCEEDED", success = false, info = "" },
+	{ result_code = "INVALID_ID", success = false, info = "" },
+	{ result_code = "DUPLICATE_NAME", success = false, info = "" },
+	{ result_code = "APPLICATION_NOT_REGISTERED", success = false, info = "" },
+	{ result_code = "OUT_OF_MEMORY", success = false, info = "" },
+	{ result_code = "TOO_MANY_PENDING_REQUESTS", success = false, info = "" },
+	{ result_code = "GENERIC_ERROR", success = false, info = "" },
+	{ result_code = "TRUNCATED_DATA", success = false, info = "" },
+	{ result_code = "WARNINGS", success = true, info = "" },
+	{ result_code = "WRONG_LANGUAGE", success = true, info = "" },
+	{ result_code = "RETRY", success = true, info = "" },
+	{ result_code = "SAVED", success = true, info = "" },
+	{ result_code = "UNSUPPORTED_RESOURCE", success = true, info = "" },
+
+	}
+
+--[[ General Precondition before ATF start ]]
+commonSteps:DeleteLogsFiles()
+config.defaultProtocolVersion = 2
+
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"BACKGROUND", "FULL", "LIMITED"},"PerformAudioPassThru")
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+commonSteps:PutFile("Precondition_PutFile_With_Icon", "icon.png")
+
+function Test:Precondition_Check_audioPassThruIcon_Existence()
+  testCasesForPerformAudioPassThru.Check_audioPassThruIcon_Existence(self, "icon.png")
+end
+
+function Test:Precondition_ActivateApp()
+  testCasesForPerformAudioPassThru:ActivateAppDiffPolicyFlag(self, config.application1.registerAppInterfaceParams.appName, config.deviceMAC)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+for i = 1, #hmi_result_code do
+	Test["TestStep_PerformAudioPassThru_UI_SUCCESS_TTS_"..hmi_result_code[i].result_code] = function(self)
+      local CorIdPerformAudioPassThru= self.mobileSession:SendRPC("PerformAudioPassThru",
+	    {
+	      initialPrompt = {{text = "Makeyourchoice",type = "TEXT"}},
+	      audioPassThruDisplayText1 = "DisplayText1",
+	      audioPassThruDisplayText2 = "DisplayText2",
+	      samplingRate = "16KHZ",
+	      maxDuration = 2000,
+	      bitsPerSample = "8_BIT",
+	      audioType = "PCM",
+	      muteAudio = true,
+	      audioPassThruIcon =
+		    { 
+		      value = "icon.png",
+		      imageType = "STATIC"
+		    }
+	    })
+	  EXPECT_HMICALL("TTS.Speak",
+	    {
+	      speakType = "AUDIO_PASS_THRU",
+	      ttsChunks = {{text = "Makeyourchoice", type = "TEXT"}},
+	      appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+	    })
+	  :Do(function(_,data)
+	      self.hmiConnection:SendNotification("TTS.Started",{})
+	      
+	      local function ttsSpeakResponse()
+	        self.hmiConnection:SendError(data.id, data.method, hmi_result_code[i].result_code, "error message")
```

Comment:
@maria-mih, as you are verifying success true and false, for correct HMI response you should use if-else-end as example:
if (hmi_result_code[i].success == true) then
	  		self.hmiConnection:SendResponse(data.id, data.method, hmi_result_code[i].result_code, {}) 
	  	else
			self.hmiConnection:SendError(data.id,data.method, hmi_result_code[i].result_code, "error") 
	  	end